packages/evolution-generator/src/tests/test_conditionals_generator.py

```
@@ -9,14 +9,53 @@
+            {"value": "true", "expected": True},
+            {"value": "TRUE", "expected": True},
+            {"value": False, "expected": False},
+            {"value": "false", "expected": False},
+            {"value": "FALSE", "expected": False},
+            {"value": 0, "expected": 0},
+            {"value": "0", "expected": 0},
+            {"value": 42, "expected": 42},
+            {"value": -3, "expected": -3},
+            {"value": 3.14, "expected": 3.14},
+            {"value": "42", "expected": 42},
+            {"value": "0", "expected": 0},
```

Ruff (0.15.12)

[warning] Duplicate of test case at index 7 in `pytest.mark.parametrize`

Remove duplicate test case

(PT014)